packages/terra-site/webpack.config.js

```diff
@@ -63,6 +64,7 @@ module.exports = {
       supportedLocales: i18nSupportedLocales,
     }),
     new webpack.NamedChunksPlugin(),
+    new UglifyJSPlugin(),
```

Comment:
does this have an effect on compilation speed? I think we noticed this slowed kaiju down quite a bit. Should we have a prod/dev webpack config?

Reply:
Yeah, its really slow. I'm just pushing it up to ensure it deploys correctly then I'll move it to the prod config.